modules/web-console/e2e/testcafe/mocks/agentTasks.js

```diff
@@ -129,6 +129,24 @@ export const SIMPLE_QUERY_RESPONSE = {
     }
 };
 
+export const SIMPLE_FAILED_QUERY_RESPONSE = {
```

Comment:
Lets add test with response from "old" node that has no stackTrace.

Reply:
Added test of cause without stacktrace.